src/deploy.js

```diff
@@ -27,14 +30,15 @@ const trackProgress = function(progressData) {
   };
 };
 
-module.exports = function(progressData, context, client, storage, config, slackTemplate) {
+module.exports = function(progressData, context, client, storage, config, slackTemplate, managementClientId) {
```

Comment:
good catch, I was passing in name originally, then changed to ID and missed that config was passed in!